diff -up e-smith-hosts-2.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm.fix-speechmarks e-smith-hosts-2.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm
--- e-smith-hosts-2.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm.fix-speechmarks	2010-11-29 09:24:40.000000000 +0100
+++ e-smith-hosts-2.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm	2010-11-29 09:35:31.000000000 +0100
@@ -271,7 +271,7 @@ like($_STDOUT_, qr(<td>foo</td>), "print
 sub print_td
 {
     my ($self, $value) = @_;
-    print "<td class=\"sme-border\">$value</td>\n";
+    printf ("<td class=\"sme-border\">%s</td>\n", $value);
 }
 
 sub build_host_cgi_params {
