diff -Nur -x '*.orig' -x '*.rej' e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries mezzanine_patched_e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries
--- e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries	2007-02-19 04:57:30.000000000 -0700
+++ mezzanine_patched_e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries	2007-02-19 04:56:54.000000000 -0700
@@ -198,7 +198,7 @@
             The ethernet address is optional and causes the DHCP server to
             statically bind the local IP address to the computer with this
             ethernet address.  If specified, it must be of the form
-            "AA:BB:CC:DD:EE:FF" and must contain only thenumbers 0-9 and
+            "AA:BB:CC:DD:EE:FF" and must contain only the numbers 0-9 and
             the letters A-F.
         </trans>
     </entry>
