--- e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries.hosttype	2008-07-08 19:19:10.000000000 +0200
+++ e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries	2008-07-08 19:22:31.000000000 +0200
@@ -35,6 +35,12 @@
         </trans>
     </entry>
     <entry>
+        <base>HOSTTYPE</base>
+        <trans>
+            Location
+        </trans>
+    </entry>
+    <entry>
         <base>LOCAL_IP</base>
         <trans>
             Local IP
