diff -Nur -x '*.orig' -x '*.rej' e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries mezzanine_patched_e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries
--- e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries	2008-02-10 11:06:14.000000000 +1100
+++ mezzanine_patched_e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries	2008-02-10 11:04:20.000000000 +1100
@@ -2,18 +2,12 @@
     <!-- vim: ft=xml
     -->
     <entry>
-        <base>FORM_TITLE</base>
+        <base>Hostnames and addresses</base>
         <trans>
             Hostnames and addresses
         </trans>
     </entry>
     <entry>
-        <base>OPERATION_STATUS_REPORT</base>
-        <trans>
-            Operation status report
-        </trans>
-    </entry>
-    <entry>
         <base>UNABLE_TO_OPEN_CONFIGDB</base>
         <trans>Unable to open configuration database</trans>
     </entry>
@@ -35,28 +29,12 @@
         </trans>
     </entry>
     <entry>
-        <base>NAME</base>
-        <trans>
-            Hostname
-        </trans>
-    </entry>
-    <entry>
         <base>HOSTNAME</base>
         <trans>
             Hostname
         </trans>
     </entry>
     <entry>
-        <base>ACTION</base>
-        <trans>Action</trans>
-    </entry>
-    <entry>
-        <base>Location</base>
-        <trans>
-            LOCATION
-        </trans>
-    </entry>
-    <entry>
         <base>LOCAL_IP</base>
         <trans>
             Local IP
@@ -75,18 +53,6 @@
         </trans>
     </entry>
     <entry>
-        <base>MODIFY</base>
-        <trans>
-            Modify
-        </trans>
-    </entry>
-    <entry>
-        <base>REMOVE</base>
-        <trans>
-            Remove
-        </trans>
-    </entry>
-    <entry>
         <base>CURRENT_HOSTNAMES_FOR_DOMAIN</base>
         <trans>
             Current list of hostnames for {$domain}.
@@ -123,12 +89,6 @@
             Create a new hostname for this server</trans>
     </entry>
     <entry>
-        <base>CREATE</base>
-        <trans>
-            Create
-        </trans>
-    </entry>
-    <entry>
         <base>HOSTNAME_VALIDATION_ERROR</base>
         <trans>
             Error: unexpected characters in host name: 
@@ -234,19 +194,6 @@
         </trans>
     </entry>
     <entry>
-        <base>HOSTNAME_LABEL</base>
-        <trans>
-            Hostname
-        </trans>
-    </entry>
-    <entry>
-        <base>HOSTNAME_DESCRIPTION</base>
-        <trans>
-            The hostname must contain only letters, numbers, and
-            hyphens, and must start with a letter or number. 
-        </trans>
-    </entry>
-    <entry>
         <base>DOMAIN</base>
         <trans>
             Domain
@@ -279,30 +226,6 @@
         </trans>
     </entry>
     <entry>
-        <base>HOSTTYPE</base>
-        <trans>
-            Location
-        </trans>
-    </entry>
-    <entry>
-        <base>ADD</base>
-        <trans>
-            Add
-        </trans>
-    </entry>
-    <entry>
-        <base>SAVE</base>
-        <trans>
-            Save
-        </trans>
-    </entry>
-    <entry>
-        <base>OPERATION_STATUS_REPORT</base>
-        <trans>
-            Operation status report
-        </trans>
-    </entry>
-    <entry>
         <base>LOCATION</base>
         <trans>
             Location
@@ -326,56 +249,11 @@
         </trans>
     </entry>
     <entry>
-        <base>NEXT</base>
-        <trans>
-            Next
-        </trans>
-    </entry>
-    <entry>
-        <base>SELF</base>
-        <trans>
-            Self
-        </trans>
-    </entry>
-    <entry>
-        <base>REMOTE</base>
-        <trans>
-            Remote
-        </trans>
-    </entry>
-    <entry>
-        <base>LOCAL</base>
-        <trans>
-            Local
-        </trans>
-    </entry>
-    <entry>
         <base>NO_HOSTS_FOR_THIS_DOMAIN</base>
         <trans>
             There are no hosts for this domain.
         </trans>
     </entry>
-
-    <entry>
-        <base>Configuration</base>
-        <trans>Configuration</trans>
-    </entry>
-    <entry>
-        <base>Hostnames and addresses</base>
-        <trans>Hostnames and addresses</trans>
-    </entry>
-    <entry>
-        <base>NO</base>
-        <trans>No</trans>
-    </entry>
-    <entry>
-        <base>YES</base>
-        <trans>Yes</trans>
-    </entry>
-    <entry>
-        <base>IP_ADDR</base>
-        <trans>IP Address</trans>
-    </entry>
     <entry>
         <base>ADDR_IN_DHCP_RANGE</base>
         <trans>Address is inside the DHCP assigned dynamic range</trans>
@@ -401,8 +279,4 @@
         <trans>Error: This IP address is not on any of our local
             networks.</trans>
     </entry>
-    <entry>
-        <base>COMMENT</base>
-        <trans>Comment</trans>
-    </entry>
 </lexicon>
diff -Nur -x '*.orig' -x '*.rej' e-smith-hosts-1.14.0/root/etc/e-smith/web/functions/hostentries mezzanine_patched_e-smith-hosts-1.14.0/root/etc/e-smith/web/functions/hostentries
--- e-smith-hosts-1.14.0/root/etc/e-smith/web/functions/hostentries	2008-02-10 11:06:14.000000000 +1100
+++ mezzanine_patched_e-smith-hosts-1.14.0/root/etc/e-smith/web/functions/hostentries	2008-02-10 10:58:22.000000000 +1100
@@ -78,7 +78,7 @@
 
 
 __DATA__
-<form title="FORM_TITLE" header="/etc/e-smith/web/common/head.tmpl"
+<form title="Hostnames and addresses" header="/etc/e-smith/web/common/head.tmpl"
     footer="/etc/e-smith/web/common/foot.tmpl">
     <page name="First" pre-event="print_status_message()">
         <subroutine src="print_hosts_tables()" />
@@ -123,7 +123,7 @@
     <page name="Remote" pre-event="turn_off_buttons()" post-event="goto_confirm()">
         <description>REMOTE_PAGE_DESCRIPTION</description>
         <field type="text" id="global_ip" validation="ip_number">
-            <label>IP_ADDR</label>
+            <label>IP_ADDRESS</label>
         </field>
         <subroutine src="print_button('NEXT')" />
     </page>
diff -Nur -x '*.orig' -x '*.rej' e-smith-hosts-1.14.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm mezzanine_patched_e-smith-hosts-1.14.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm
--- e-smith-hosts-1.14.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm	2008-02-10 11:06:14.000000000 +1100
+++ mezzanine_patched_e-smith-hosts-1.14.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm	2008-02-10 10:50:02.000000000 +1100
@@ -267,7 +267,7 @@
     my $fm = shift;
 
     print qq(<tr><td colspan=2>) . $fm->localise('HOSTNAME_DESCRIPTION') . qq(</td></tr>);
-    print qq(<tr><td class="sme-noborders-label">) . $fm->localise('HOSTNAME_LABEL') . qq(</td>\n);
+    print qq(<tr><td class="sme-noborders-label">) . $fm->localise('HOSTNAME') . qq(</td>\n);
 
     my $h = $fm->{cgi}->param('name');
 
