diff -Nur -x '*.orig' -x '*.rej' e-smith-hosts-1.13.3/root/etc/e-smith/events/actions/purge-domain mezzanine_patched_e-smith-hosts-1.13.3/root/etc/e-smith/events/actions/purge-domain
--- e-smith-hosts-1.13.3/root/etc/e-smith/events/actions/purge-domain	2005-07-19 08:34:27.000000000 +1000
+++ mezzanine_patched_e-smith-hosts-1.13.3/root/etc/e-smith/events/actions/purge-domain	2006-01-24 16:01:36.426388918 +1100
@@ -59,7 +59,7 @@
 
     next unless $domain;
 
-    next if exists $domain_names{$nym};
+    next if exists $domain_names{$domain};
 
     $nym->delete;
 }
