diff -Nur -x '*.orig' -x '*.rej' e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries mezzanine_patched_e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries
--- e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries	2007-07-06 11:36:36.000000000 -0400
+++ mezzanine_patched_e-smith-hosts-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/hostentries	2007-07-05 13:42:56.000000000 -0400
@@ -75,12 +75,6 @@
         </trans>
     </entry>
     <entry>
-        <base>GLOBAL_IP</base>
-        <trans>
-            Global IP
-        </trans>
-    </entry>
-    <entry>
         <base>ETHERNET_ADDRESS</base>
         <trans>
             Ethernet address
@@ -242,15 +236,6 @@
         </trans>
     </entry>
     <entry>
-        <base>DIDNT_ENTER_GLOBAL_IP</base>
-        <trans>
-            Error: You did not enter a Global IP value:
-            A value is required in order to publish
-            this entry globally. Did not create
-            hostname.
-        </trans>
-    </entry>
-    <entry>
         <base>CREATE_REMOTE_HOST_TITLE</base>
         <trans>
             Create a new hostname referring to a remote host
@@ -282,19 +267,6 @@
         </trans>
     </entry>
     <entry>
-        <base>PUBLISH_GLOBALLY_LABEL</base>
-        <trans>
-            Publish globally
-        </trans>
-    </entry>
-    <entry>
-        <base>PUBLISH_GLOBALLY_SUBSCRIBED</base>
-        <trans>
-            If you select "publish globally" this hostname will
-            automatically be made available throughout the Internet. 
-        </trans>
-    </entry>
-    <entry>
         <base>LOCAL_PAGE_DESCRIPTION</base>
         <trans>
             Please enter the following additional details for a local
@@ -309,14 +281,6 @@
         </trans>
     </entry>
     <entry>
-        <base>GLOBAL_IP_DESCRIPTION</base>
-        <trans>
-            Since you selected "publish globally", this hostname will
-            automatically be made available throughout the Internet,
-            pointing to the specified "global IP" address. 
-        </trans>
-    </entry>
-    <entry>
         <base>SUCCESSFULLY_DELETED</base>
         <trans>
             Successfully deleted host.
@@ -399,12 +363,6 @@
         </trans>
     </entry>
     <entry>
-        <base>GLOBAL</base>
-        <trans>
-            Global
-        </trans>
-    </entry>
-    <entry>
         <base>LOCAL</base>
         <trans>
             Local
diff -Nur -x '*.orig' -x '*.rej' e-smith-hosts-1.14.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm mezzanine_patched_e-smith-hosts-1.14.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm
--- e-smith-hosts-1.14.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm	2007-07-06 11:36:36.000000000 -0400
+++ mezzanine_patched_e-smith-hosts-1.14.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/hostentries.pm	2007-07-05 13:41:48.000000000 -0400
@@ -50,7 +50,6 @@
     mac_address_or_blank
     not_in_dhcp_range
     print_save_or_add_button
-    publish_checkbox
     not_taken
     must_be_local
 );
@@ -605,26 +604,6 @@
         "OK";
 }
 
-sub publish_checkbox
-{
-    my $self = shift;
-    my $q = $self->{cgi};
-    my $publish = $q->param('publish_globally');
-    my %boxparams = (
-        -type => 'checkbox',
-        -name => 'publish_globally'
-    );
-    $boxparams{checked} = 1 if $publish;
-
-    print $q->Tr($q->td({-class => 'sme-noborders-label'},
-    $self->localise('PUBLISH_GLOBALLY')),
-    $q->td({-class => 'sme-noborders-content'},
-    $q->input(\%boxparams)));
-    $q->delete('publish_globally');
-
-    return undef;
-}
-
 =head2 not_taken
 
 This function checks the local ip address being set in the panel, and 
