diff -Nur -x '*.orig' -x '*.rej' e-smith-hosts-1.13.1/root/etc/e-smith/events/actions/conf-migrate-hosts mezzanine_patched_e-smith-hosts-1.13.1/root/etc/e-smith/events/actions/conf-migrate-hosts
--- e-smith-hosts-1.13.1/root/etc/e-smith/events/actions/conf-migrate-hosts	2005-07-18 16:20:26.295695176 -0600
+++ mezzanine_patched_e-smith-hosts-1.13.1/root/etc/e-smith/events/actions/conf-migrate-hosts	2005-07-18 16:18:52.317978248 -0600
@@ -54,7 +54,7 @@
 }
 
 use esmith::HostsDB;
-my $hdb = esmith::HostsDB->open_ro();
+my $hdb = esmith::HostsDB->open();
 
 =begin testing
 
@@ -195,17 +195,17 @@
     foreach my $host ( @ShortHostList )
     {
 	$properties{'InternalIP'} = 
-		db_get_prop (\%hosts, $host, 'InternalIP') || '';
+		$hdb->get_prop ($host, 'InternalIP') || '';
 
 	$properties{'ExternalIP'} = 
-		db_get_prop (\%hosts, $host, 'ExternalIP') || '';
+		$hdb->get_prop ($host, 'ExternalIP') || '';
 
 	$properties{'MACAddress'} = 
-		db_get_prop (\%hosts, $host, 'MACAddress') || '';
+		$hdb->get_prop ($host, 'MACAddress') || '';
 
 	my $FQDN = "$host.$domain";
 
-	next if (defined db_get (\%hosts, $FQDN));
+	next if (defined $hdb->get ($FQDN));
 
 	# If defined as host 'self', change to new format
 	if (($properties{'InternalIP'} eq 'self') && 
@@ -231,7 +231,8 @@
             warn "$host has no InternalIP or ExternalIP!\n";
         }
 
-	db_set (\%hosts, $FQDN, 'host', \%properties);
+        $properties{'type'} = 'host';
+	$hdb->new_record($FQDN, \%properties);
     }
 }
 
@@ -244,7 +245,7 @@
 foreach my $key ( @FQDNHostList )
 {
     # Skip the hosts if already in the new format
-    next if (defined db_get_prop (\%hosts, $key, 'HostType'));
+    next if (defined $hdb->get_prop ($key, 'HostType'));
 
     my %properties = (
 	Visibility => 'Local'
@@ -260,9 +261,9 @@
 	$properties{'error'} = 'Host has no matching domain in system';
     }
 
-    $properties{'InternalIP'} = db_get_prop (\%hosts, $key, 'InternalIP') || '';
-    $properties{'ExternalIP'} = db_get_prop (\%hosts, $key, 'ExternalIP') || '';
-    $properties{'MACAddress'} = db_get_prop (\%hosts, $key, 'MACAddress') || '';
+    $properties{'InternalIP'} = $hdb->get_prop ($key, 'InternalIP') || '';
+    $properties{'ExternalIP'} = $hdb->get_prop ($key, 'ExternalIP') || '';
+    $properties{'MACAddress'} = $hdb->get_prop ($key, 'MACAddress') || '';
 
     # If defined as host 'self', change to new format
     if ($properties{'InternalIP'} eq 'self')
@@ -278,14 +279,15 @@
 	$properties{'ExternalIP'} = '';
     }
 
-    db_set (\%hosts, $key, 'host', \%properties);
+    $properties{'type'} = 'host';
+    $hdb->new_record($key, \%properties);
 }
 
 
 # Remove the old single host entries
 foreach my $host ( @ShortHostList )
 {
-    db_delete (\%hosts, $host);
+    $hdb->get($host)->delete;
 }
 
 exit (0);
