diff -Nur -x '*.orig' -x '*.rej' e-smith-hosts-1.13.1/root/etc/e-smith/events/actions/delete-hosts mezzanine_patched_e-smith-hosts-1.13.1/root/etc/e-smith/events/actions/delete-hosts
--- e-smith-hosts-1.13.1/root/etc/e-smith/events/actions/delete-hosts	2003-05-30 14:23:37.000000000 -0600
+++ mezzanine_patched_e-smith-hosts-1.13.1/root/etc/e-smith/events/actions/delete-hosts	2005-07-18 01:26:51.664952104 -0600
@@ -41,26 +41,17 @@
 my $event = $ARGV [0];
 
 # Look for invalid hosts in the hostsdb.
-my @domain_names = map { $_->{key} } $domainsdb->domains;
+my %domain_names = map { $_->{key} => 1 } $domainsdb->domains;
 
 # We want to check each host and see if it belongs to a domain that we are
 # currently managing. If not, delete it.
 foreach my $host ($hostsdb->hosts)
 {
     my ($hostpart, $domainpart) = split (/\./, $host->{key}, 2);
-    my $found = FALSE;
-    foreach my $domain_name (@domain_names)
-    {
-	if ($domainpart eq $domain_name)
-	{
-	    $found = TRUE;
-	    last;
-	}
-    }
-    if (! $found)
-    {
-	$host->delete;
-    }
+
+    next if exists $domain_names{$domainpart};
+
+    $host->delete;
 }
 
 exit 0;
